dockerize/Dockerfile

```diff
@@ -102,13 +86,25 @@ COPY dockerize/supervisord.conf /etc/
 # Copy the rest of B2Share files
 #
 
-WORKDIR /eudat/b2share
-ADD webui webui
-WORKDIR /eudat/b2share/webui
-RUN node_modules/webpack/bin/webpack.js -p
-
 WORKDIR /eudat
 ADD . b2share
+RUN rm -rf /eudat/b2share/webui
```

Comment:
I assume that this done because the COPY command on line 107 fails to write (or overwrite) the /eudat/b2share/webui ?
Maybe add a line comment?
Something like "# Remove webui code brought in with 'ADD . b2share' since webui build is done in its own stage.

Reply:
Comments have been added to make the stages more clear